repos/keg-cli/src/tasks/tap/run.js

```diff
@@ -63,8 +61,8 @@ module.exports = {
       },
       satisfy: {
         alias: [ 'sat', 'ensure' ],
-        description: 'Will check if required images are built, and build them in necessary.',
-        example: "keg ${ task } ${ action } --satisfy false",
+        description: 'Will check if required docker images are pulled and built. Will then pull and build images as needed',
+        example: `keg tap run --no-satisfy`,
```

Comment:
did not know adding a `no-` before an option gives it a falsy value by default.
thats cool

Reply:
word